src/collaborative_platform/apps/help/views.py

```diff
@@ -0,0 +1,226 @@
+from dal import autocomplete
+
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.db.models import Q
+from django.forms import inlineformset_factory
+from django.http import HttpResponse, HttpRequest
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+
+from apps.core.models import Profile
+from apps.files_management.models import Directory
+from apps.views_decorators import objects_exists, user_has_access
+from apps.projects.loggers import ProjectsLogger
+
+from .forms import ContributorForm, ProjectEditForm
+from .models import Project, Contributor
+
```

Comment:
Same for this code. Why is it copied to _help_ app?

Reply:
Hum, my mistake. I initially forked from it and left those two bits behind.